packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -195,12 +195,13 @@ LatLngBounds gmLatLngBoundsTolatLngBounds(gmaps.LatLngBounds latLngBounds) {
   );
 }
 
-CameraPosition _gmViewportToCameraPosition(gmaps.GMap map) {
+CameraPosition _gmViewportToCameraPosition(gmaps.Map map) {
   return CameraPosition(
-    target: gmLatLngToLatLng(map.center ?? _nullGmapsLatLng),
-    bearing: map.heading?.toDouble() ?? 0,
-    tilt: map.tilt?.toDouble() ?? 0,
-    zoom: map.zoom?.toDouble() ?? 0,
+    target:
+        gmLatLngToLatLng(map.isCenterDefined() ? map.center : _nullGmapsLatLng),
```

Comment:
(I've [sent a14n a question](https://github.com/a14n/dart-google-maps/pull/135#issuecomment-2246748984) about the `isXxxDefined()` APIs instead of making `.xxx` nullable)

Reply:
The reason is that once a map is initialized, there's a lot of things that will never be null (like center, latlng, and others) so it's faster to not have to check!